### ISSUE TITLE
Limit queue

### ISSUE THREAD

**Issue by aterriblegamer** [yermak/AudioBookConverter#70]
I am in the process of converting a large series and this issue came up in the workflow. 

Adding a book to the queue starts the process immediately. There is no option to limit the number of jobs processed at a time. I would like to be able to tell the app to only work on 2 items at a time and "queue" the rest for processing.

**Comment by yermak** [comment]
@aterriblegamer,
Good idea. In current architecture it will be a bit tricky to implement, but possible.
As a work around you could pause jobs just once you add them, to allow converter progress on the first ones.

**Comment by baloodusudouest** [comment]
Great idea, i have also a lot of audiobook to converter, and when i add them one by one i have some error.
If we can have maybe 2 button or only one to configure like send to the queue or send to direct transcode, something like this.

Thanks a lot for you work!
